src/CreateEventTypeView/CreateEventTypeView.js

```diff
@@ -5,103 +5,185 @@ import {Container} from "react-grid-system";
-                </Container>
-            </div>
-        );
-    }
+		if (!event.target.value) {
+			updatedAttributes.splice(currentAttributeIndex, 1);
+			this.setState({attributes: updatedAttributes});
+			return;
+		}
+
+		updatedAttributes[currentAttributeIndex].value = event.target.value;
+
+		if (parseInt(currentAttributeId, 10) === this.state.lastAttributeId) {
+			const newAttributeId = this.nextAttributeId;
+			this.nextAttributeId += 1;
+			const newAttribute = {id: `${newAttributeId}`, value: ''};
+			this.setState({
+				attributes: updatedAttributes.concat([newAttribute]),
+				lastAttributeId: newAttributeId
+			});
+		}
+		else {
+			this.setState({attributes: updatedAttributes});
+		}
+	}
+
+	render() {
+		const optionalActions = this.props.deleteQueryResponse;
+		if(optionalActions && optionalActions.fulfilled) {
+			window.history.back();
+			return null;
+		}
+		return (
+			<div>
+				<Header title="Create New Event Type"/>
+				<Container>
+					{optionalActions && optionalActions.rejected &&
+						<ErrorMessage message={optionalActions.reason} />
+					}
+					<form>
+						<div className={css(AppStyles.dFlex, AppStyles.elementMarginTop)}>
+							<Col>
+								<Row>
+									<Col>
+										<TextField
+											floatingLabelText="Event Type Name"
+											fullWidth={true}
+											value={this.state.eventTypeName}
+											onChange={this.handleChangeEventTypeName}
+											required={true}
+											errorText={this.state.eventTypeNameErrorText}/>
+									</Col>
+								</Row>
+								<Row>
+									<Col>
+										<TextField
+											floatingLabelText="Timestamp Attribute"
+											fullWidth={true}
+											value={this.state.eventTypeTimestampAttribute}
+											onChange={this.handleChangeEventTypeTimestampAttribute}
+											errorText={this.state.eventTypeTimestampAttributeErrorText}/>
+									</Col>
+								</Row>
+								<Row className={css(AppStyles.elementMarginTop)}>
+									<Col md={4}> Attributes: </Col>
+								</Row>
+								<Row>
+									<Col>
+										{this.state.attributes.map((attribute) =>
+											<TextField
+												key={attribute.id}
+												id={attribute.id}
+												value={attribute.value}
+												hintText="Attribute Name"
+												onChange={this.onInputChange.bind(this)}
+												fullWidth={true}/>
+										)}
+									</Col>
+								</Row>
+							</Col>
+						</div>
+					</form>
+					<div className={css(AppStyles.textAlignCenter)}>
+						<RaisedButton
+							label="Abort"
+							icon={<IconCancel/>}
+							className={css(AppStyles.marginAllSites)}
+							secondary={true}/>
+						<RaisedButton
+							label="Save"
+							icon={<IconSave/>}
+							onClick={this.submitForm}
+							className={css(AppStyles.marginAllSites)}
+							primary={true}/>
+					</div>
+				</Container>
+			</div>
+		);
+	}
 }
 
-export default CreateEventTypeView;
+export default ConnectionComponent.argosConnector()(props => ({
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "props". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)